tests/test_lane.py

```diff
@@ -4172,8 +4182,26 @@ def works(self, _db, facets, *args, **kwargs):
 
         # Each lane's adapted faceting object was then passed into
         # works().
-        eq_("Custom facets for Lane 1.", child1.works_called_with)
-        eq_("Custom facets for Lane 2.", child2.works_called_with)
+        eq_((pagination, "Custom facets for Lane 1."), child1.works_called_with)
+
+        eq_((pagination, "Custom facets for Lane 2."), child2.works_called_with)
+
+        # If no pagination object is passed in (the most common case),
+        # a new Pagination object is created based on the featured lane
+        # size for the library.
+        groups = list(
+            parent._groups_for_lanes(self._db, relevant, queryable, None,
+                                     facets)
+        )
+
+        (ignore1, pagination, ignore2) = parent._featured_works_with_lanes_called_with
+        assert isinstance(pagination, Pagination)
```

Comment:
Should we care about consistency and try to use `eq_` as much as possible? For example, does it make sense to replace this code with the following?
```python
eq_(True, isinstance(pagination, Pagination))
```

Reply:
I'm not terribly concerned about this.